go-fuzz-build/main.go

```diff
@@ -582,3 +618,28 @@ func main() {
 	dep.Main(target.%v)
 }
 `
+
+var mainSrc_libFuzzer = `
+package main
+
+import (
+	"C"
+	"unsafe"
+	target "%v"
+	dep "go-fuzz-dep"
+)
+
+
+//export gofuzz_Run
+func gofuzz_Run(input []byte) {
+	target.%v(input)
+}
+
+//export gofuzz_Init
+func gofuzz_Init(coverTabPtr unsafe.Pointer, coverTabSize uint64) {
+	dep.Initialize(coverTabPtr, coverTabSize)
+}
+
+func main() {
```

Comment:
Is this required if we are building a c-archive?

Reply:
If I remove it, I get

```
runtime.main_main·f: function main is undeclared in the main package
```

Reply:
ack